mindjunkies/forums/tests/test_views.py

```
@@ -0,0 +1,79 @@
+        self.assertEqual(response.status_code, 200)
+        self.assertTemplateUsed(response, 'forums/forum_home.html')
+
+    def test_forum_thread_view(self):
+        self.client.login(username='testuser', password='password')
+        response = self.client.get(reverse('forum_thread', kwargs={'course_slug': self.course.slug, 'module_id': self.module.id}))
+        self.assertEqual(response.status_code, 200)
+        self.assertTemplateUsed(response, 'forums/forum_threads.html')
+
+    def test_forum_thread_details_view(self):
+        self.client.login(username='testuser', password='password')
+        response = self.client.get(reverse('forum_thread_details', kwargs={'course_slug': self.course.slug, 'topic_slug': self.forum_topic.slug}))
```

Ruff (0.8.2)

Line too long (146 > 119)

(E501)